src/cr/cube/cube.py

```diff
@@ -773,6 +793,50 @@ def _flat_values(self):
         ).flatten()
 
 
+class _OverlapMeasure(_BaseMeasure):
+    """Statistical overlap values from a cube-response."""
+
+    @lazyproperty
+    def missing_count(self):
+        """Numeric value representing count of missing rows in response."""
+        return self._cube_dict["result"]["measures"]["overlap"].get("n_missing", 0)
```

Comment:
do we need missing counts for overlaps? If yes this property is not used anywhere. So I think that the business logic at line 636 (`_Measures` -> `missing_count`) should be augmented.

Reply:
we don't need it afaics